### ISSUE TITLE
Provide pointer for Add Thing by URL feature

### ISSUE THREAD

**Issue by rei-vilo** [mozilla-iot/esp8266-wot#1]
Could you please provide a pointer to the **Add Thing by URL** feature?

Thank you!

**Comment by rei-vilo** [comment]
Connecting a browser directly to the ESP8266 displays

```
{
 "name": "ESP8266 LED",
  "type": "onOffSwitch",
  "href": "/",
  "properties": {
    "on": {
      "type": "boolean",
      "href": "/properties/on"
    }
  },
  "actions": {},
  "events": {}
}
```

**Comment by hobinjk** [comment]
The Add Thing by URL feature doesn't exist yet but I plan to mock up an initial implementation for it soon. Until then, its tracking issue is https://github.com/mozilla-iot/gateway/issues/69. Sorry for the confusion!

**Comment by rei-vilo** [comment]
Thank you!

**Comment by julienrat** [comment]
Hi ! Same for me ! impatient to have this feature ! ;-)

**Comment by rei-vilo** [comment]
Does [release 0.3.1](https://github.com/mozilla-iot/gateway/releases/tag/0.3.1) include the `Add Thing by URL` feature?

**Comment by hobinjk** [comment]
Add Thing by URL is present on the current master branch of the Gateway and will be available in our upcoming 0.4.0 release. Thanks for waiting!
